src/main/java/cc/polyfrost/oneconfig/utils/JsonUtils.java

```diff
@@ -39,6 +41,35 @@ public final class JsonUtils {
      */
     public static final JsonParser PARSER = new JsonParser();
 
+    /**
+     * Runs the provided consumer if the given string is a valid JSON string.
+     * @param json The JSON string to check.
+     * @param consumer The consumer to run if the string is a valid JSON.
+     * @return Whether the string is a valid JSON string.
+     */
+    public static boolean ifValid(String json, Consumer<JsonElement> consumer) {
+        try {
+            consumer.accept(PARSER.parse(json));
+            return true;
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    /**
+     * Checks whether the given string is a valid JSON.
+     * @param json The string to check.
+     * @return Whether the string is a valid JSON.
+     */
+    public static boolean isValid(String json) {
+        try {
+            PARSER.parse(json);
```

Comment:
ok but surely there is a lighter way to do this

Reply:
wrong